map/place_page_info.hpp

```diff
@@ -268,4 +275,20 @@ class Info : public osm::MapObject
   std::string m_localsUrl;
   LocalsStatus m_localsStatus = LocalsStatus::NotAvailable;
 };
+
+namespace rating
+{
+enum Impress
+{
+  None,
+  Horrible,
+  Bad,
+  Normal,
+  Good,
+  Excellent
+};
+
+static Impress GetImpress(float const rawRating);
```

Comment:
Sorry, what's the reason to declare functions as static in header?

Reply:
sorry, it's my fault